librecomp/include/librecomp/overlays.hpp

```diff
@@ -2,19 +2,20 @@
 #define __RECOMP_OVERLAYS_H__
 
 #include <cstdint>
+#include <cstddef>
 #include "sections.h"
 
 namespace recomp {
     namespace overlays {
         struct overlay_section_table_data_t {
             SectionTableEntry* code_sections;
-            size_t num_code_sections;
-            size_t total_num_sections;
+            std::size_t num_code_sections;
+            std::size_t total_num_sections;
         };
 
         struct overlays_by_index_t {
             int* table;
-            size_t len;
+            std::size_t len;
```

Comment:
Actually this is the only place we use `std::size_t` instead of just plainly `size_t`, so I think it would be better to change those to just `size_t`

Reply:
`std::size_t` is also used below in `void register_patches(const char* patch_data, std::size_t patch_size` so if we're going based on majority, there were more `std::size_t` than just `size_t`

Reply:
I mean in the repo overall, not just this system

Reply:
I seeee! ok yeah sure, I'll update them.